schema.graphql

```diff
@@ -20,4 +20,10 @@ type Product @table @export {
 type Traits @table @export {
   id: ID @primaryKey
   traits: [String]
+}
+
+type PersonalizeCache @table @export {
+  id: ID @primaryKey
```

Comment:
I tried making the key the primaryKey since it needs to be unique, but I'm thinking a UUID is expected since a string did not work. Like this:

type PersonalizeCache @table @export {
  key: ID @primaryKey
  content: String
}

or this:
type PersonalizeCache @table @export {
  key: String @primaryKey
  content: String
}

(Neither technique worked when I went to .create or .put or .post a new record